test/postman.test.ts

```diff
@@ -0,0 +1,115 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { assert } from 'chai';
+import { Contract, ContractFactory } from 'ethers';
+import { starknet, network, ethers } from 'hardhat';
+import {
+  StarknetContractFactory,
+  StarknetContract,
+  HttpNetworkConfig,
+} from 'hardhat/types';
+
+describe.only('Postman', () => {
+  let L2contractFactory: StarknetContractFactory;
```

Comment:
Did you leave variable names capitalized on purpose?

Reply:
I used capitalized names for factories and fixed contracts to be start with lowercase [e7c5890](https://github.com/Shard-Labs/starknet-hardhat-example/pull/19/commits/e7c58908ad72f88d30d87173c430dfbec67c0ffb)